update-assets-docker.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -ex
+
+# Default values
+IMAGE_NAME="simulator"
+REUSE_IMAGE=false
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --reuse-image)
+      REUSE_IMAGE=true
+      ;;
+    *)
+      echo "Invalid argument: $1"
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+# Check if the Docker image exists locally
+if ! "$REUSE_IMAGE" || ! docker image inspect "$IMAGE_NAME" &> /dev/null; then
+  echo "Docker image $IMAGE_NAME does not exist or --reuse-image flag not provided. Building..."
+  docker build -t $IMAGE_NAME -f simulator/Dockerfile .
+else
+  echo "Docker image $IMAGE_NAME exists, and --reuse-image flag is set. Skipping build."
+fi
+
+for crate_name in "faucet" "radiswap" "flash_loan" "genesis_helper" "metadata" "test_environment" "global_n_owned" "kv_store"
```

Comment:
We're missing `max_transaction`, which is added recently.

Reply:
noted. will add